scss/_patterns_badge.scss

```diff
@@ -0,0 +1,34 @@
+@import 'settings';
+
+@mixin vf-p-badge {
+  %vf-badge {
+    @extend %x-small-text;
+    background-color: $color-dark;
+    border-radius: 1rem;
+    box-sizing: content-box;
+    color: $color-x-light;
+    margin-bottom: 0;
+    max-width: 4ch;
+    overflow: hidden;
+    padding: 0 $spv--x-small;
+  }
+
+  .p-badge,
+  .p-badge--negative {
+    @extend %vf-badge;
+  }
+
+  .p-badge--negative {
```

Comment:
I have some doubts about calling this one the "negative".

While it does use red colour, I have a feeling that it's not really meant to mean anything specifically negative (like an error), we also don't have any other (positive, caution) compared to other components that use "negative" modifiers.

On one hand it's good that it's consistent, but also seems strange when looking just at the Badge component in isolation.

@rayito @lyubomir-popov Is there any specific use case for using the red variant of the badge? Should it use the same "negative" name as our other "negative" labels, buttons, etc?

Reply:
Let's keep it for now, for consistency. We may create another alias name if needed.